src/lib/utils/__tests__/renderWithLoadProgress-tests.tsx

```diff
@@ -1,63 +1,26 @@
-import { mount } from "enzyme"
-import * as React from "react"
-import { Image, Text, View } from "react-native"
-import { Artwork, badQuery, query, renderToString } from "../../tests/__tests__/MockRelayRendererFixtures"
-import { MockRelayRenderer } from "../../tests/MockRelayRenderer"
-import { renderUntil } from "../../tests/renderUntil"
+import Spinner from "lib/Components/Spinner"
+import { extractText } from "lib/tests/extractText"
+import React from "react"
+import { Text } from "react-native"
+import ReactTestRenderer from "react-test-renderer"
+import renderWithLoadProgress from "../renderWithLoadProgress"
 
-jest.unmock("react-relay")
+describe(renderWithLoadProgress, () => {
+  it(`renders a spinner while the graphqls are loaeding`, () => {
+    const result = renderWithLoadProgress(() => null, {})({ error: null, props: null, retry: () => null })
 
-describe("MockRelayRenderer", () => {
```

Comment:
this file was previously just a duplicate of the test file for MockRelayRender (!!)